traffic_ops/testing/api/v2/servers_test.go

```diff
@@ -101,17 +101,23 @@ func UpdateTestServers(t *testing.T) {
 	hostName := firstServer.HostName
 	// Retrieve the server by hostname so we can get the id for the Update
 	resp, _, err := TOSession.GetServerByHostName(hostName)
+	originalHostname := resp[0].HostName
```

Comment:
Test cases to test the following:
1. Change in hostname updated the field hostName but doesn't impact xmppid
2. Also, any change to xmppid, the Update() ignores the change.